### PR TITLE
fix: `AttributeDistributionTotal` in event emit

### PR DESCRIPTION
**Pull request by yaruwangway**
## Description

Closes: #720 

fix the attribute `AttributeDistributionTotal` value in `FeeDistribution` event emit. `AttributeDistributionTotal` was always empty before.

Fix

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

* [x] Included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
* [ ] Added `!` to the type prefix if state-machine breaking change (i.e., requires coordinated upgrade)
* [x] Confirmed this PR does not introduce changes requiring state migrations, OR migration code has been added to consumer and/or provider modules
* [x] Targeted the correct branch (see [PR Targeting](https://github.com/cosmos/interchain-security/blob/main/CONTRIBUTING.md#pr-targeting))
* [ ] Provided a link to the relevant issue or specification
* [ ] Followed the guidelines for [building SDK modules](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/building-modules)
* [ ] Included the necessary unit and integration [tests](https://github.com/cosmos/interchain-security/blob/main/CONTRIBUTING.md#testing)
* [x] Added a changelog entry to `CHANGELOG.md`
* [ ] Included comments for [documenting Go code](https://blog.golang.org/godoc)
* [ ] Updated the relevant documentation or specification
* [x] Reviewed "Files changed" and left comments if necessary 
* [x] Confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

* [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
* [ ] confirmed `!` in the type prefix if API or client breaking change
* [ ] confirmed this PR does not introduce changes requiring state migrations, OR confirmed migration code has been added to consumer and/or provider modules
* [ ] confirmed all author checklist items have been addressed
* [ ] reviewed state machine logic
* [ ] reviewed API design and naming
* [ ] reviewed documentation is accurate
* [ ] reviewed tests and test coverage

### CODE REVIEW

File: x/ccv/consumer/keeper/distribution.go [hunk 1]
```diff
@@ -153,7 +152,7 @@ func (k Keeper) SendRewardsToProvider(ctx sdk.Context) error {
 				sdk.NewAttribute(ccv.AttributeDistributionCurrentHeight, strconv.Itoa(int(currentHeight))),
 				sdk.NewAttribute(ccv.AttributeDistributionNextHeight, strconv.Itoa(int(currentHeight+k.GetBlocksPerDistributionTransmission(ctx)))),
 				sdk.NewAttribute(ccv.AttributeDistributionFraction, (k.GetConsumerRedistributionFrac(ctx))),
-				sdk.NewAttribute(ccv.AttributeDistributionTotal, fpTokens.String()),
+				sdk.NewAttribute(ccv.AttributeDistributionTotal, allBalances.String()),
```
Comment (smarshall-spitzbart) [inline x/ccv/consumer/keeper/distribution.go:155]: would something like `AttributeTotalFeePool` be a better name here? Likely not important tho

**Review by smarshall-spitzbart — APPROVED** [review]
Good stuff!

**Review by MSalopek — APPROVED** [review]
LGTM!

Backporting to `v2` and `v3` using labels.

File: x/ccv/consumer/keeper/distribution.go (hunk 1 above)
Comment (yaruwangway) [reply x/ccv/consumer/keeper/distribution.go:155]: i can change it here, it is a var name, so not api breaking right?

Comment (smarshall-spitzbart) [reply x/ccv/consumer/keeper/distribution.go:155]: I'm not sure what API breaking would mean in this context, if you change the value string then the event will change (which seems ok? I don't think changing an event is state breaking but I could be wrong), if you just change the var naming then it's a pure refactor. Again not super important, so feel free to ignore my comment if we're not confident how the change would affect things

**Pull request merged**